tests/test_preprocess.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import sys
+import os
+sys.path.append("..") #Add top level directory to python path for imports
+from corn_app import preprocess
+from PIL import Image
+
+class TestKernelCounter(unittest.TestCase):
```

Comment:
```TestKernelCoutner``` is the class name?

Reply:
I'm not sure what you mean. Does this matter? Or is it because test_filters.py has the same class?

Reply:
test_filters.py only has that class name because I forgot to change it when I changed kernel_counter.py to filters.py. The code will run, but the class name doesn't match.